sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusMessage.cs

```diff
@@ -33,26 +34,25 @@ public class ServiceBusMessage
         /// </summary>
         /// <param name="body">The payload of the message as a string.</param>
```

Comment:
Should we call out that this is a UTF-8 string for clarity?

Reply:
Actually, this is already in the summary for the ctor.